src/main/java/core/basesyntax/service/FileReaderServiceImpl.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderServiceImpl implements FileReaderService {
+    @Override
+    public List<String> readFile(String filepath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader br = new BufferedReader(new FileReader(filepath))) {
+            String line;
+            while ((line = br.readLine()) != null) {
+                lines.add(line);
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Using `e.printStackTrace()` is not recommended for production code as it can clutter the console and is not user-friendly. Consider using a logging framework to log the exception or rethrow it as a custom exception to handle it more gracefully.